pkgs/development/python-modules/pyscf/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildPythonPackage, lib, fetchFromGitHub, libcint, libxc, xcfun, blas,
+  numpy, scipy, h5py, python3
+} :
+
+buildPythonPackage rec {
+  pname = "pyscf";
+  version = "1.7.6.post1";
```

Comment:
```suggestion
  version = "1.7.6";
```

Reply:
@sheepforce Was there a particular reason for using a "non-release" commit?

Reply:
This is not exactly 1.7.6 but actually the post release as labeled on PyPi. To be consistent I think it might be better to keep the version that is also on PyPi?

Reply:
XCFun does not work with 1.7.6. 1.7.6.post1 addresses this and only this. This is basically the release version as on PyPi. https://pypi.org/project/pyscf/1.7.6.post1/
It just does not have a git tag.